src/analytics.ts

```diff
@@ -0,0 +1,18 @@
+declare var gtag: any;
+
+export const EVENT_ACTION = "action";
+
+export const trackEvent = (
+  name: string,
+  category: string,
+  label?: string,
+  value?: number,
+) => {
+  if (gtag) {
+    gtag("event", name, {
```

Comment:
how to properly load (make ts aware of gtag if available) from https://github.com/excalidraw/excalidraw/blob/master/public/index.html#L99